site2/docs/security-policy-and-supported-versions.md

```diff
@@ -14,6 +14,12 @@ https://pulsar.apache.org/docs/en/security-overview/.
 The Pulsar community will announce security vulnerabilities and how to mitigate them on the [users@pulsar.apache.org](mailto:users@pulsar.apache.org).
 For instructions on how to subscribe, please see https://pulsar.apache.org/contact/.
 
+## Reporting Vulnerabilities
+
+The Pulsar community follows the ASF [vulnerability handling process](https://apache.org/security/#vulnerability-handling).
+
+To report a new vulnerability you have discovered please follow the [ASF vulnerability reporting process](https://apache.org/security/#reporting-a-vulnerability). Besides, you can send one plain-text email for each vulnerability to [private@pulsar.apache.org](mailto:private@pulsar.apache.org).
```

Comment:
Are users allowed to send emails to `security@apache.org` as well?

Reply:
Yes. See also the links in the content:

* https://www.apache.org/security/projects.html
* https://apache.org/security/#vulnerability-handling
* https://apache.org/security/#reporting-a-vulnerability

Reply:
If users are allowed / encouraged to send emails to `security@apache.org` and `private@pulsar.apache.org`

1. Why only add `private@pulsar.apache.org` to guide (in this PR)?

2. When reporting a Pulsar security issue, is there any difference between `security@apache.org` and `private@pulsar.apache.org`? Or user can report to any of them, they are the same?

Reply:
> Why only add private@pulsar.apache.org to guide (in this PR)?

It starts with `Besides, ...`. Actually, we can remove the possibility to report to private@ to make it easy. A vulnerability reported to security@ will be carried to the PMC (private@) if necessary.

Reply:
```suggestion
To report a new vulnerability you have discovered please follow the [ASF vulnerability reporting process](https://apache.org/security/#reporting-a-vulnerability).
```

... like this.

Reply:
Updated.